app/src/androidTest/resources/binary/surrogates/surrogates_1

```diff
@@ -0,0 +1,107 @@
+# To neutralize GA scripts. The goal is to provide the minimal API
```

Comment:
So these files are being served up by an endpoint for licensing issues.  To avoid any problems can you create mock versions of these files, please?